google/generativeai/generative_models.py

```diff
@@ -489,8 +497,8 @@ class ChatSession:
         history: A chat history to initialize the object with.
     """
 
-    _USER_ROLE = "user"
-    _MODEL_ROLE = "model"
+    _USER_ROLE = _USER_ROLE
```

Comment:
Why do you need to set these on the class? We already have the global `_*ROLE` constants

Reply:
Global roles were added later.

Removed it to use Global ones.